hvplot/converter.py

```diff
@@ -337,6 +337,22 @@ def __init__(self, data, x, y, kind=None, by=None, use_index=True,
                     "following plot types: %r" % (self.kind, self._geo_types))
             from cartopy import crs as ccrs
             from geoviews.util import project_extents
+
+            if isinstance(projection, basestring):
+                all_crs = [proj for proj in dir(ccrs) if
+                           callable(getattr(ccrs, proj)) and
+                           proj not in ['ABCMeta', 'CRS'] and
+                           proj[0].isupper() or
+                           proj == 'GOOGLE_MERCATOR']
+                if projection in all_crs and projection != 'GOOGLE_MERCATOR':
+                    projection = getattr(ccrs, projection)()
+                elif projection == 'GOOGLE_MERCATOR':
+                    projection = getattr(ccrs, projection)
```

Comment:
I don't quite know what's happening with GOOGLE_MERCATOR here.  Could it maybe be handled by checking if it's callable, and only calling callable things?